ansible/roles/datadog/tasks/remove_integrations.yml

```diff
@@ -12,7 +12,7 @@
     path: /etc/dd-agent/checks.d/couchdb2.py
     state: absent
   notify: restart datadog
-  when: inventory_hostname != couchdb2_first_host or not DATADOG_INTEGRATIONS_ENABLED|default(DATADOG_ENABLED)
+  when: True
```

Comment:
could just remove the `when` clause completely

Reply:
Oops. yeah. I had that around to get the old one removed when I was deploying. I think we can just remove this whole block